config/application-test.yml

```diff
@@ -13,6 +13,21 @@ spring:
     hibernate.ddl-auto: create
     show-sql: false
     properties.hibernate.default_schema: PUBLIC
+  security:
+    oauth2:
+      client:
+        provider:
+          wisvchconnect:
+            issuer-uri: https://connect.ch.tudelft.nl
+        registration:
+          wisvchconnect:
+            client-id: localhost-spring-dev
```

Comment:
Deze wijziging betekent ook dat het Kubernetes manifest moet worden aangepast, bijv. `SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_WISVCHCONNECT_CLIENT_ID` voor de client id.

Reply:
Een PR voor flux beheer is geopened met de geupdate values